mail_message_icon_demo/models/calendar_event.py

```diff
@@ -1,5 +1,4 @@
 import pytz
-
```

Comment:
Pre commits in our internal project works different from here (i.e. imports from odoo were always separated by one empty line). I guess we should unify them. Added topic for our internal dev meeting